src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+      ">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card-image"
+        src="./images/imac.jpeg"
+        alt="imac"
+      >
+      <h2 class="card-title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card-code">
+        Product code: 195434
+      </p>
+      <div class="stars">
+        <ul class="stars__list stars__list--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+        <p class="stars__text">Reviews: 5</p>
+      </div>
+      <div class="card-price">
+        <p class="card-price__text">Price:</p>
+        <p class="card-price__number">$2,199</p>
```

Comment:
_Checklist №6 [BEM] - Don't make BEM structure too complicated. You don't need to create separate blocks for anything other than card itself and stars._

Consider making 'card-price'  a card element:shushing_face: